### PR TITLE
Remove all cases of Total Playtime from API files

### PR DESCRIPTION
**Pull request by atl128**
Links to https://github.com/osuAkatsuki/hanayo/pull/8.

### CODE REVIEW

**Comment by atl128** [comment]
would recommend flushing all records of total playtime from the db after this is merged as they will have no use

File: app/v1/user.go [hunk 1]
```diff
@@ -301,39 +300,39 @@ LIMIT 1
 
 		&b.Icon, &b.Name, &can, &show,
 
-		&r.Stats[0].STD.RankedScore, &r.Stats[0].STD.TotalScore, &r.Stats[0].STD.PlayCount, &r.Stats[0].STD.PlayTime, &r.Stats[0].STD.Total_PlayTime,
+		&r.Stats[0].STD.RankedScore, &r.Stats[0].STD.TotalScore, &r.Stats[0].STD.PlayCount, &r.Stats[0].STD.PlayTime
```
Comment (cmyui) [inline app/v1/user.go:303]:
```suggestion
		&r.Stats[0].STD.RankedScore, &r.Stats[0].STD.TotalScore, &r.Stats[0].STD.PlayCount, &r.Stats[0].STD.PlayTime,
```

File: app/v1/user.go [hunk 2]
```diff
@@ -301,39 +300,39 @@ LIMIT 1
...
 		&r.Stats[0].STD.ReplaysWatched, &r.Stats[0].STD.TotalHits,
 		&r.Stats[0].STD.Accuracy, &r.Stats[0].STD.PP, &r.Stats[0].STD.MaxCombo,
 
-		&r.Stats[0].Taiko.RankedScore, &r.Stats[0].Taiko.TotalScore, &r.Stats[0].Taiko.PlayCount, &r.Stats[0].Taiko.PlayTime, &r.Stats[0].Taiko.Total_PlayTime,
+		&r.Stats[0].Taiko.RankedScore, &r.Stats[0].Taiko.TotalScore, &r.Stats[0].Taiko.PlayCount, &r.Stats[0].Taiko.PlayTime
```
Comment (cmyui) [inline app/v1/user.go:307]:
```suggestion
		&r.Stats[0].Taiko.RankedScore, &r.Stats[0].Taiko.TotalScore, &r.Stats[0].Taiko.PlayCount, &r.Stats[0].Taiko.PlayTime,
```

File: app/v1/user.go [hunk 3]
```diff
@@ -301,39 +300,39 @@ LIMIT 1
...
 		&r.Stats[0].Taiko.ReplaysWatched, &r.Stats[0].Taiko.TotalHits,
 		&r.Stats[0].Taiko.Accuracy, &r.Stats[0].Taiko.PP, &r.Stats[0].Taiko.MaxCombo,
 
-		&r.Stats[0].CTB.RankedScore, &r.Stats[0].CTB.TotalScore, &r.Stats[0].CTB.PlayCount, &r.Stats[0].CTB.PlayTime, &r.Stats[0].CTB.Total_PlayTime,
+		&r.Stats[0].CTB.RankedScore, &r.Stats[0].CTB.TotalScore, &r.Stats[0].CTB.PlayCount, &r.Stats[0].CTB.PlayTime
```
Comment (cmyui) [inline app/v1/user.go:311]:
```suggestion
		&r.Stats[0].CTB.RankedScore, &r.Stats[0].CTB.TotalScore, &r.Stats[0].CTB.PlayCount, &r.Stats[0].CTB.PlayTime,
```

File: app/v1/user.go [hunk 4]
```diff
@@ -301,39 +300,39 @@ LIMIT 1
...
 		&r.Stats[0].CTB.ReplaysWatched, &r.Stats[0].CTB.TotalHits,
 		&r.Stats[0].CTB.Accuracy, &r.Stats[0].CTB.PP, &r.Stats[0].CTB.MaxCombo,
 
-		&r.Stats[0].Mania.RankedScore, &r.Stats[0].Mania.TotalScore, &r.Stats[0].Mania.PlayCount, &r.Stats[0].Mania.PlayTime, &r.Stats[0].Mania.Total_PlayTime,
+		&r.Stats[0].Mania.RankedScore, &r.Stats[0].Mania.TotalScore, &r.Stats[0].Mania.PlayCount, &r.Stats[0].Mania.PlayTime
```
Comment (cmyui) [inline app/v1/user.go:315]:
```suggestion
		&r.Stats[0].Mania.RankedScore, &r.Stats[0].Mania.TotalScore, &r.Stats[0].Mania.PlayCount, &r.Stats[0].Mania.PlayTime,
```

File: app/v1/user.go [hunk 5]
```diff
@@ -301,39 +300,39 @@ LIMIT 1
...
 		&r.Stats[1].CTB.ReplaysWatched, &r.Stats[1].CTB.TotalHits,
 		&r.Stats[1].CTB.Accuracy, &r.Stats[1].CTB.PP, &r.Stats[1].CTB.MaxCombo,
 
-		&r.Stats[1].Mania.RankedScore, &r.Stats[1].Mania.TotalScore, &r.Stats[1].Mania.PlayCount, &r.Stats[1].Mania.PlayTime, &r.Stats[1].Mania.Total_PlayTime,
+		&r.Stats[1].Mania.RankedScore, &r.Stats[1].Mania.TotalScore, &r.Stats[1].Mania.PlayCount, &r.Stats[1].Mania.PlayTime
 		&r.Stats[1].Mania.ReplaysWatched, &r.Stats[1].Mania.TotalHits,
 		&r.Stats[1].Mania.Accuracy, &r.Stats[1].Mania.PP, &r.Stats[1].Mania.MaxCombo,
 
-		&r.Stats[2].STD.RankedScore, &r.Stats[2].STD.TotalScore, &r.Stats[2].STD.PlayCount, &r.Stats[2].STD.PlayTime, &r.Stats[2].STD.Total_PlayTime,
+		&r.Stats[2].STD.RankedScore, &r.Stats[2].STD.TotalScore, &r.Stats[2].STD.PlayCount, &r.Stats[2].STD.PlayTime
```
Comment (cmyui) [inline app/v1/user.go:335]:
```suggestion
		&r.Stats[2].STD.RankedScore, &r.Stats[2].STD.TotalScore, &r.Stats[2].STD.PlayCount, &r.Stats[2].STD.PlayTime,
```

File: app/v1/user.go [hunk 6]
```diff
@@ -301,39 +300,39 @@ LIMIT 1
...
 		&r.Stats[1].Taiko.ReplaysWatched, &r.Stats[1].Taiko.TotalHits,
 		&r.Stats[1].Taiko.Accuracy, &r.Stats[1].Taiko.PP, &r.Stats[1].Taiko.MaxCombo,
 
-		&r.Stats[1].CTB.RankedScore, &r.Stats[1].CTB.TotalScore, &r.Stats[1].CTB.PlayCount, &r.Stats[1].CTB.PlayTime, &r.Stats[1].CTB.Total_PlayTime,
+		&r.Stats[1].CTB.RankedScore, &r.Stats[1].CTB.TotalScore, &r.Stats[1].CTB.PlayCount, &r.Stats[1].CTB.PlayTime
 		&r.Stats[1].CTB.ReplaysWatched, &r.Stats[1].CTB.TotalHits,
 		&r.Stats[1].CTB.Accuracy, &r.Stats[1].CTB.PP, &r.Stats[1].CTB.MaxCombo,
 
-		&r.Stats[1].Mania.RankedScore, &r.Stats[1].Mania.TotalScore, &r.Stats[1].Mania.PlayCount, &r.Stats[1].Mania.PlayTime, &r.Stats[1].Mania.Total_PlayTime,
+		&r.Stats[1].Mania.RankedScore, &r.Stats[1].Mania.TotalScore, &r.Stats[1].Mania.PlayCount, &r.Stats[1].Mania.PlayTime
```
Comment (cmyui) [inline app/v1/user.go:331]:
```suggestion
		&r.Stats[1].Mania.RankedScore, &r.Stats[1].Mania.TotalScore, &r.Stats[1].Mania.PlayCount, &r.Stats[1].Mania.PlayTime,
```

File: app/v1/user.go [hunk 7]
```diff
@@ -301,39 +300,39 @@ LIMIT 1
...
 		&r.Stats[1].STD.ReplaysWatched, &r.Stats[1].STD.TotalHits,
 		&r.Stats[1].STD.Accuracy, &r.Stats[1].STD.PP, &r.Stats[1].STD.MaxCombo,
 
-		&r.Stats[1].Taiko.RankedScore, &r.Stats[1].Taiko.TotalScore, &r.Stats[1].Taiko.PlayCount, &r.Stats[1].Taiko.PlayTime, &r.Stats[1].Taiko.Total_PlayTime,
+		&r.Stats[1].Taiko.RankedScore, &r.Stats[1].Taiko.TotalScore, &r.Stats[1].Taiko.PlayCount, &r.Stats[1].Taiko.PlayTime
 		&r.Stats[1].Taiko.ReplaysWatched, &r.Stats[1].Taiko.TotalHits,
 		&r.Stats[1].Taiko.Accuracy, &r.Stats[1].Taiko.PP, &r.Stats[1].Taiko.MaxCombo,
 
-		&r.Stats[1].CTB.RankedScore, &r.Stats[1].CTB.TotalScore, &r.Stats[1].CTB.PlayCount, &r.Stats[1].CTB.PlayTime, &r.Stats[1].CTB.Total_PlayTime,
+		&r.Stats[1].CTB.RankedScore, &r.Stats[1].CTB.TotalScore, &r.Stats[1].CTB.PlayCount, &r.Stats[1].CTB.PlayTime
```
Comment (cmyui) [inline app/v1/user.go:327]:
```suggestion
		&r.Stats[1].CTB.RankedScore, &r.Stats[1].CTB.TotalScore, &r.Stats[1].CTB.PlayCount, &r.Stats[1].CTB.PlayTime,
```

File: app/v1/user.go [hunk 8]
```diff
@@ -301,39 +300,39 @@ LIMIT 1
...
 		&r.Stats[0].Mania.ReplaysWatched, &r.Stats[0].Mania.TotalHits,
 		&r.Stats[0].Mania.Accuracy, &r.Stats[0].Mania.PP, &r.Stats[0].Mania.MaxCombo,
 
-		&r.Stats[1].STD.RankedScore, &r.Stats[1].STD.TotalScore, &r.Stats[1].STD.PlayCount, &r.Stats[1].STD.PlayTime, &r.Stats[1].STD.Total_PlayTime,
+		&r.Stats[1].STD.RankedScore, &r.Stats[1].STD.TotalScore, &r.Stats[1].STD.PlayCount, &r.Stats[1].STD.PlayTime
 		&r.Stats[1].STD.ReplaysWatched, &r.Stats[1].STD.TotalHits,
 		&r.Stats[1].STD.Accuracy, &r.Stats[1].STD.PP, &r.Stats[1].STD.MaxCombo,
 
-		&r.Stats[1].Taiko.RankedScore, &r.Stats[1].Taiko.TotalScore, &r.Stats[1].Taiko.PlayCount, &r.Stats[1].Taiko.PlayTime, &r.Stats[1].Taiko.Total_PlayTime,
+		&r.Stats[1].Taiko.RankedScore, &r.Stats[1].Taiko.TotalScore, &r.Stats[1].Taiko.PlayCount, &r.Stats[1].Taiko.PlayTime
```
Comment (cmyui) [inline app/v1/user.go:323]:
```suggestion
		&r.Stats[1].Taiko.RankedScore, &r.Stats[1].Taiko.TotalScore, &r.Stats[1].Taiko.PlayCount, &r.Stats[1].Taiko.PlayTime,
```

File: app/v1/user.go [hunk 9]
```diff
@@ -301,39 +300,39 @@ LIMIT 1
...
 		&r.Stats[0].CTB.ReplaysWatched, &r.Stats[0].CTB.TotalHits,
 		&r.Stats[0].CTB.Accuracy, &r.Stats[0].CTB.PP, &r.Stats[0].CTB.MaxCombo,
 
-		&r.Stats[0].Mania.RankedScore, &r.Stats[0].Mania.TotalScore, &r.Stats[0].Mania.PlayCount, &r.Stats[0].Mania.PlayTime, &r.Stats[0].Mania.Total_PlayTime,
+		&r.Stats[0].Mania.RankedScore, &r.Stats[0].Mania.TotalScore, &r.Stats[0].Mania.PlayCount, &r.Stats[0].Mania.PlayTime
 		&r.Stats[0].Mania.ReplaysWatched, &r.Stats[0].Mania.TotalHits,
 		&r.Stats[0].Mania.Accuracy, &r.Stats[0].Mania.PP, &r.Stats[0].Mania.MaxCombo,
 
-		&r.Stats[1].STD.RankedScore, &r.Stats[1].STD.TotalScore, &r.Stats[1].STD.PlayCount, &r.Stats[1].STD.PlayTime, &r.Stats[1].STD.Total_PlayTime,
+		&r.Stats[1].STD.RankedScore, &r.Stats[1].STD.TotalScore, &r.Stats[1].STD.PlayCount, &r.Stats[1].STD.PlayTime
```
Comment (cmyui) [inline app/v1/user.go:319]:
```suggestion
		&r.Stats[1].STD.RankedScore, &r.Stats[1].STD.TotalScore, &r.Stats[1].STD.PlayCount, &r.Stats[1].STD.PlayTime,
```

**Review by cmyui — COMMENTED** [review]
thanks! just one issue - syntax errors from missing commas

**Comment by cmyui** [comment]
> would recommend flushing all records of total playtime from the db after this is merged as they will have no use

yep this is the plan - we'll make a database migration script to drop the total_playcount row from the stats tables (vn, rx, ap)

**Comment by cmyui** [comment]
thanks! both PRs have been deployed to production

**Pull request merged**